### ISSUE TITLE
AnsibleUndefinedVariable: 'dict object' has no attribute 'rabbitmq_servers'

### ISSUE THREAD

**Issue by deadghost** [cmacrae/ansible-sensu#8]
`ansible 2.0.1.0`
`ubuntu 14.04`

Running the play with defaults:

```
- name: Install and config sensu stack.
  hosts: prod
  become: true
  become_method: sudo
  roles:
    - role: cmacrae.sensu
```

`$ ansible-playbook prod/sensu.yml`

Things of note:

```
TASK [cmacrae.sensu : Deploy the Sensu client SSL cert/key] ********************
[DEPRECATION WARNING]: Skipping task due to undefined Error, in the future this 
will be a fatal error.. This feature will be removed in a future release. 
Deprecation warnings can be disabled by setting deprecation_warnings=False in 
ansible.cfg.

TASK [cmacrae.sensu : Deploy Sensu client RabbitMQ configuration] **************
fatal: [kimsufi]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'rabbitmq_servers'"}
```

The warning `undefined Error` is not helpful so I'm stuck as to how to resolve it and whether it is causing the failure down the chain. Any ideas?

**Comment by cmacrae** [comment]
The warning is likely due to an Ansible version mismatch of the version the code is intended for and the version of Ansible you are running. This role is supported on the Ansible 2.0 release only.

As for the actual error, the variable `rabbitmq_servers` is not defined.
Have you followed the documentation and ensured you have a group `rabbitmq_servers` defined within your inventory?

EDIT: My apologies, I've just seen your information detailing your Ansible version - I'll have to look into the warning message.

**Comment by cmacrae** [comment]
@deadghost Are you still experiencing this issue? Please see my comment above

**Comment by deadghost** [comment]
Yeah, I got further along. I missed a good portion of the documentation the first time. 'Deploying the Stack to a Single Node' clarified things.

I am now stuck on the `Ensure Redis is Running` step. It results in:

`fatal: [kimsufi]: FAILED! => {"changed": false, "failed": true, "invocation": {"module_args": {"arguments": "", "enabled": true, "name": "redis", "pattern": null, "runlevel": "default", "sleep": null, "state": "started"}, "module_name": "service"}, "msg": "**no service or tool found for: redis**"}`

This is probably due to `ubuntu 14.04` not being a supported platform because the previous step probably uses systemd which is not available on 14.04.
